src/Rule/NumberHandler.php

```diff
@@ -18,6 +18,8 @@
  * The format of the number must match the regular expression specified in {@see Number::$integerPattern}
```

Comment:
Such details can be removed in the handler, the link for corresponding rule is enough in my opinion (or a shortened version of description). But I prefer the link more.